### PR TITLE
Modified activation functions according to tf.keras.activations.tanh #25827

### PR DESCRIPTION
**Pull request by shashvatshahi1998**
I modified tanh(),sigmoid() and exp() functions

### CODE REVIEW

**Review by alextp — CHANGES_REQUESTED** [review]
This PR looks like it has a few unrelated changes. Can you have it have one change only?

**Comment by shashvatshahi1998** [comment]
i have updated it

**Comment by shashvatshahi1998** [comment]
in this commit i have just added details to exp() function, if you will ask me i will also update tanh ,linear and sigmoid functions.

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -175,6 +175,21 @@ def sigmoid(x):
...
+  Arguments:
+      x = input tesor
+      y = e^x
+  Returns:
+      A tensor
+
+  Input:[-0.5 -0.1  0.   0.1  0.5] 
+  Output:[0.60653067 0.9048374  1.         1.105171   1.6487212 ]
+  '''
+>>>>>>> c5ffa1c... modified some activation functions
```
Comment (arp95) [inline tensorflow/python/keras/activations.py]: This seems to be a merge conflict. Please resolve this.

**Comment by kyscg** [comment]
There's a better fix in #26603 

Think we can close this @alextp @rthadur
